src/main/java/io/cdap/plugin/servicenow/sink/model/SchemaResponse.java

```diff
@@ -16,44 +16,21 @@
 
 package io.cdap.plugin.servicenow.sink.model;
 
+import java.util.List;
+
 /**
  * Model class for Schema Response from Schema API
  */
 public class SchemaResponse {
-  private String label;
-  private String exampleValue;
-  private String internalType;
-  private String name;
-
-  public String getLabel() {
-    return label;
-  }
-
-  public void setLabel(String label) {
-    this.label = label;
-  }
-
-  public String getExampleValue() {
-    return exampleValue;
-  }
-
-  public void setExampleValue(String exampleValue) {
-    this.exampleValue = exampleValue;
-  }
-
-  public String getInternalType() {
-    return internalType;
-  }
-
-  public void setInternalType(String internalType) {
-    this.internalType = internalType;
-  }
+  
+  private List<ServiceNowSchemaField> result;
 
-  public String getName() {
-    return name;
+  public void setResult(List<ServiceNowSchemaField> result) {
```

Comment:
similar comment to above, set all fields once in a constructor and don't modify state.

Reply:
All fields are not being set in the constructor. cbfb212c77242fe61b2ef14c690a9079b6a9e22f